src/components/Cards/assignee-badge.tsx

```diff
@@ -1,29 +1,131 @@
+import { SyntheticEvent, useState } from "react";
 import Image from "next/image";
+import { useRouter } from "next/router";
+import { RepairStatus } from "@prisma/client";
+
+import Button from "@/components/Button";
+import Circle from "@/components/Cards/circle";
+import VolunteerCard from "@/components/Cards/volunteer-card";
+import Modal from "@/components/Modal";
+import { useRepairers } from "@/hooks/events";
+import { useUpdateRepairRequest } from "@/hooks/repair-request";
 
 type Props = {
-  firstName?: string;
-  lastName?: string;
-  avatar?: string;
+  firstName: string | undefined;
+  lastName: string | undefined | null;
+  avatar?: string | undefined;
+  assignedTo?: string;
+  repairRequestId?: string;
```

Comment:
Should this repairRequestId be optional?

Reply:
I presume it should? bc there the page `/manage/events/[id]/volunteers` does not pass in a repair request ID when using this component.